src/lesson5/tv/Tv.java

```diff
@@ -0,0 +1,65 @@
+package lesson5.tv;
+
+import java.util.Objects;
+
+public class Tv {
+    private int currentChannel = 1;
+    private int currentSound = 0;
+    private final String manufacturer;
+    private boolean onOrOfTv = false;
+
+    public Tv(String manufacturer) {
+        this.manufacturer = manufacturer;
+    }
+
+    public int getCurrentChannel() {
+        return currentChannel;
+    }
+
+    public void setCurrentChannel(int currentChannel) {
+        this.currentChannel = currentChannel;
+    }
+
+    public int getCurrentSound() {
+        return currentSound;
+    }
+
+    public void setCurrentSound(int currentSound) {
+        this.currentSound = currentSound;
+    }
+
+    public String getManufacturer() {
+        return manufacturer;
+    }
+
+    public boolean isOnOrOfTv() {
+        return onOrOfTv;
+    }
+
+    public void setOnOrOfTv(boolean onOrOfTv) {
```

Comment:
make two methods: enable and disable. or call it `changeState`

Reply:
ok